commonlib-build.properties

```diff
@@ -18,13 +18,14 @@ inject.jar=lib/guice-3.0/javax.inject.jar
 log4j.jar=lib/log4j/log4j-1.2.17.jar
 commonsio.jar=lib/commons-io-2.4/commons-io-2.4.jar
 derby.jar=lib/Derby/derby.jar
+influx.jar=lib/influxdb-java-2.5.jar
```

Comment:
Good question, fundamentally no, and the reason is that the common lib is in theory common, it shouldn't depend on the competition project at all. For instance the common lib was used in the xbotedu project you were using before.